model/sharing/sharing.go

```diff
@@ -1568,6 +1568,7 @@ func (s *Sharing) checkSharingTreesConsistency(inst *instance.Instance, ownerDoc
 						"member":    m.Domain,
 						"ownerDoc":  ownerDoc,
 						"memberRev": memberDoc.Rev(),
+						"memberID":  memberDoc.M["_id"].(string),
```

Comment:
There's an `ID()` method to do that :)
```suggestion
						"memberID":  memberDoc.ID(),
```

Reply:
thanks, updated